### ISSUE TITLE
add "data source" field to radionuclide_info.json

### ISSUE THREAD

**Issue by KrisThielemans** [UCL/STIR#1339]
It would be nice to have the original source as part of the database. This should be possible by just adding a field "data source" specifying the URL or free-form field.

e.g.
```
        {
            "name":  "^18^Fluorine",
            "decays": [                
                {  "data_source" : "someURL",
                   "modality": "PET",
                   "keV": 511,
                   "branching_ratio": 0.9686,
                   "half_life": 6584.04
                }
            ]
        },
```
We think this needs no code changes, but would break backwards compatibility in some sense, so should be done before releasing 6.0.

_Originally posted by @KrisThielemans in https://github.com/UCL/STIR/issues/1338#issuecomment-1903740075_

**Comment by KrisThielemans** [comment]
Fixed by #1369
